### DISCUSSION TITLE
Show an image on iOS notifications

### DISCUSSION THREAD

**Discussion by scottjodoin** [TobiasBuchholz/Plugin.Firebase#469] [General]
I would like to display an image in my notification, ideally to replace the app icon. Is that work in the scope of this project? If not, what is the path forward for me to be able to implement it?

I have found the following resoures:

Apple guide on customizing the notification appearance:
https://developer.apple.com/documentation/usernotificationsui/customizing-the-appearance-of-notifications

Creating iOS extensions in Xamarin in Maui
https://vladislavantonyuk.github.io/articles/Extend-.NET-MAUI-application-with-iOS-Extensions/

Creating push notifications with images in Xamarin
https://stackoverflow.com/questions/54412303/xamarin-ios-cannot-display-photo-in-push-notification

**Comment by AdamEssenmacher** [comment]
This project is a wrapper over the native iOS Firebase SDKs. AFAIK, the native iOS FCM SDK has nothing to do with displaying images in notifications, which means it's out of the current scope of this project.

**Comment by TobiasBuchholz** [comment]
Take a look at the comment in this related issue https://github.com/TobiasBuchholz/Plugin.Firebase/issues/100#issuecomment-1347210588.
